packages/ui/src/image-uploader.tsx

```
@@ -0,0 +1,131 @@
+  uploadedFile,
+  onReset,
+  onUpload,
+}: ImageUploaderProps) => {
+  const [isDragging, setIsDragging] = useState(false);
+  const [files, setFiles] = useState<File[]>([]);
+  const [previewUrl, setPreviewUrl] = useState<string | null>(null);
+
+  useEffect(() => {
+    if (files.length > 0) {
+      const file = files[0];
+      if (file && file.type.startsWith('image')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)